xbmc/guilib/guiinfo/SystemGUIInfo.cpp

```diff
@@ -320,6 +320,32 @@ bool CSystemGUIInfo::GetLabel(std::string& value, const CFileItem *item, int con
       StringUtils::ToCapitalize(value);
       return true;
 #endif
+    case SYSTEM_SUPPORTED_HDR_TYPES:
+    {
+      if (CServiceBroker::GetWinSystem()->IsHDRDisplay())
+      {
+        // Assumes HDR10 minimum requirement for HDR
+        std::string types = "HDR10";
+
+        const CHDRCapabilities caps = CServiceBroker::GetWinSystem()->GetDisplayHDRCapabilities();
+
+        if (caps.SupportsHLG())
+          types += ", HLG";
+        if (caps.SupportsHDR10Plus())
+          types += ", HDR10+";
+        if (caps.SupportsDolbyVision())
+          types += ", Dolby Vision";
+
+        value = types;
+      }
+      else
+      {
+        // None
```

Comment:
Hopefully I am not late to the party but I think this can be improved. Infolabels are resources that are exposed to multiple subsystems: GUI, skins, python, etc. They should contain values that can be easily processed.
If you make this dependent on the user Kodi configured language, you break this rule - i.e. it's hard to know, e.g. from python, that no HDR mode is supported.
I'd suggest to return an empty string here and translate that in GUIWindowSystemInfo using a ternary operator or something similar (depending on whether the infolabel isEmpty())

Reply:
Should be fixed now.
Thank you.